src/4D_api.cpp

```diff
@@ -291,6 +291,15 @@ similarly, but prefers the 2D resp. 3D interfaces if available.
             if( iBest < 0 ) {
                 break;
             }
+            if( iRetry > 0 ) {
+                const int oldErrno = proj_errno_reset(P);
+                if (proj_log_level(P->ctx, PJ_LOG_TELL) >= PJ_LOG_DEBUG) {
```

Comment:
Is there always going to be a valid `oldErrno`?

Reply:
hopefully if we get here, errno should have been set to something sensible previously.